src/python_testing/TC_MCORE_FS_1_3.py

```diff
@@ -0,0 +1,181 @@
+#
+#    Copyright (c) 2024 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+# This test requires a TH_SERVER application that returns UnsupportedAttribute when reading UniqueID from BasicInformation Cluster. Please specify with --string-arg th_server_app_path:<path_to_app>
+
+import logging
+import os
+import random
+import signal
+import subprocess
+import time
+import uuid
+
+import chip.clusters as Clusters
+from chip import ChipDeviceCtrl
+from chip.interaction_model import Status
+from matter_testing_support import MatterBaseTest, TestStep, async_test_body, default_matter_test_main, type_matches
+from mobly import asserts
+
+
+class TC_MCORE_FS_1_3(MatterBaseTest):
+    @async_test_body
+    async def setup_class(self):
+        super().setup_class()
+        self.device_for_th_eco_kvs = None
+        self.device_for_th_eco_port = 5543
+        self.app_process_for_th_eco = None
+
+        self.device_for_dut_eco_kvs = None
+        self.device_for_dut_eco_port = 5544
+        self.app_process_for_dut_eco = None
+
+        # Create a second controller on a new fabric to communicate to the server
+        new_certificate_authority = self.certificate_authority_manager.NewCertificateAuthority()
+        new_fabric_admin = new_certificate_authority.NewFabricAdmin(vendorId=0xFFF1, fabricId=2)
+        paa_path = str(self.matter_test_config.paa_trust_store_path)
+        self.TH_server_controller = new_fabric_admin.NewController(nodeId=112233, paaTrustStorePath=paa_path)
+
+    def teardown_class(self):
+        if self.app_process_for_dut_eco is not None:
+            logging.warning("Stopping app with SIGTERM")
+            self.app_process_for_dut_eco.send_signal(signal.SIGTERM.value)
+            self.app_process_for_dut_eco.wait()
+        if self.app_process_for_th_eco is not None:
+            logging.warning("Stopping app with SIGTERM")
+            self.app_process_for_th_eco.send_signal(signal.SIGTERM.value)
+            self.app_process_for_th_eco.wait()
+
+        os.remove(self.device_for_dut_eco_kvs)
+        if self.device_for_th_eco_kvs is not None:
+            os.remove(self.device_for_th_eco_kvs)
+        super().teardown_class()
+
+    async def create_device_for_dut_ecosystem(self):
```

Comment:
This is also commissioning it, right? Similar to `create_and_commission_device_for_th_ecosystem`

Reply:
Yes... they originally started off different but now they are identical I will merge the two